app/src/main/java/com/shuttl/packagetest/MainActivity.kt

```diff
@@ -52,12 +52,16 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
         requestLocationPermission()
 
-        val intent = Intent(this, LocationPingService::class.java)
-        intent.action = "STOP"
+        if (!BuildConfig.BUILD_TYPE.equals("debug"))
```

Comment:
which flavour you will be using?

Reply:
We are using Debug . 
I have created one. more Build Type **QA** .